docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -509,7 +509,72 @@
   "properties" : {
     "plan" : {
       "type" : "object",
-      "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:JobPlanInfo:RawJson"
+      "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:JobPlanInfo:Plan",
```

Comment:
Have you been able to check if the result json looks the same? I mean, we should be sure that the old client can safely read the json that is created by this new version(at least Autopilot reads this raw json)

Reply:
Thanks. the returned response doesn't change (except null fields which are added)
I updated the description with example, wdyt?

Reply:
Yes, it looks good. Two new fields should not be a problem. It's important that the old fields remain the same, and they do.

Reply:
@akalash  @Efrat19 thanks for the explanations. From the discussion - this seems reasonable. I am curious as to what the 2 new null fields mean, do we gain anything from them? I am thinking we can now differentiate between a field that does not exist an one that exists but has a null value. Do we have tests that can drive each of these cases? Or is this not an important aspect to this - as they are effectively treated the same?

Reply:
For nulls, should we mark the fields with @JsonInclude(Include.NON_NULL) (I guess that should hide them if they're null)

Reply:
Thanks. Addressed in a followup PR https://github.com/apache/flink/pull/26488